CONTRIBUTING.md

```diff
@@ -190,15 +190,40 @@ that can help you.
 
 ## Content contributors
 
-### Creating a news post
+### Creating a blog post
 
-*Coming soon! We are still figuring out exactly how this will look*
+Blog posts must be created in the ``conda-dot-org/blog`` directory with a file naming convention of ``YYYY-MM-DD-title.md``. 
 
+Let's say you're announcing a cool new project you've recently released. Your file would be named something like ``2023-04-22-cool-project-v1-release.md``.
 
-### Creating a blog post
+conda.org blog posts support the following metadata:
 
-*Coming soon! We are still figuring out exactly how this will look*
+```
+title: The title of your blog.
```

Comment:
What other metadata do we support? Authors? Description? Other stuff?

Reply:
@kathatherine , take a look at the Docusaurus docs for this:

- https://docusaurus.io/docs/blog#adding-posts
- https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog#markdown-front-matter